log-native/src/main/java/org/ballerinalang/stdlib/log/Utils.java

```diff
@@ -18,124 +18,97 @@
+        StringBuilder message = new StringBuilder();
+        for (Map.Entry<BString, Object> entry : logRecord.entrySet()) {
+            String key = entry.getKey().toString();
+            String value;
+            switch (entry.getKey().toString()) {
+                case "time":
+                    value = entry.getValue().toString();
+                    break;
+                case "level":
+                    value = entry.getValue().toString();
+                    if (value.equals("INFO") || value.equals("WARN")) {
+                        value = value + " ";
+                    }
+                    break;
+                case "module":
+                    value = entry.getValue().toString();
+                    if (value.equals("")) {
+                        value = "\"" + value + "\"";
+                    }
+                    break;
+                default:
+                    if (entry.getValue() instanceof BString) {
+                        value = "\"" + escape(entry.getValue().toString()) + "\"";
+                    } else {
+                        value = entry.getValue().toString();
+                    }
+                    break;
             }
+            message.append(key).append(" = ").append(value).append(" ");
         }
+        err.println(message);
     }
 
     /**
-     * Sets the global log level.
+     * Get the name of the current module.
      *
-     * @param logLevel log level
+     * @return module name
      */
-    public static void setGlobalLogLevelExtern(BString logLevel) {
-        ballerinaUserLogLevel = BLogLevel.toBLogLevel(logLevel.getValue());
-        loggerLevels.put(GLOBAL_PACKAGE_PATH, BLogLevel.toBLogLevel(logLevel.getValue()));
-    }
-
-    /**
-     * Sets the module log level.
-     *
-     * @param module module
-     * @param logLevel log level
-     */
-    public static void setModuleLogLevelExtern(BString module, BString logLevel) {
-        loggerLevels.put(module.getValue(), BLogLevel.toBLogLevel(logLevel.getValue()));
+    public static BString getModuleName() {
+        String className = Thread.currentThread().getStackTrace()[5].getClassName();
+        String[] pkgData = className.split("\\.");
+        if (pkgData.length > 1) {
+            String module = IdentifierUtils.decodeIdentifier(pkgData[1]);
+            return StringUtils.fromString(pkgData[0] + "/" + module);
+        }
+        return StringUtils.fromString(".");
     }
 
     /**
-     * Checks if module log level has been enabled.
+     * Get the current local time.
      *
-     * @return true if module log level has been enabled, false if not.
+     * @return current local time in RFC3339 format
      */
-    private static boolean isModuleLogLevelEnabled() {
-        return loggerLevels.size() > 1;
+    public static BString getCurrentTime() {
+        return StringUtils.fromString(
+                new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX")
```

Comment:
Don't we need to this `yyyy-MM-dd'T'HH:mm:ss.SSSXXX` format?

Reply:
have milliseconds range presentation

Reply:
Fixed in caeb8cc